src/pages/BlockInfoPage.tsx

```diff
@@ -160,15 +162,15 @@ const BlockInfoPage = () => {
 
       {blockInfo?.mainChain ? (
         !txLoading && (!txList || (txListStatus && txListStatus !== 200)) ? (
-          <InlineErrorMessage message="An error occured while fetching transactions" code={txListStatus} />
+          <InlineErrorMessage message={t('An error occured while fetching transactions')} code={txListStatus} />
         ) : (
           <>
             <SecondaryTitle>Transactions</SecondaryTitle>
             <Table main hasDetails scrollable isLoading={txLoading}>
               {txList && txList && (
                 <>
                   <TableHeader
-                    headerTitles={['', 'Hash', 'Inputs', '', 'Outputs', 'Total Amount', '']}
+                    headerTitles={['', t('Hash'), t('Inputs'), '', t('Outputs'), t('Total Amount'), '']}
```

Comment:
Idea: we could modify the `TableHeader` component to apply the translation function there (in order to just pass strings here). WDYT?

Reply:
done :rocket: much nicer